### PR TITLE
Use the same order of items as in Encore

### PR DESCRIPTION
**Pull request by alexwlchan**
For https://github.com/wellcomecollection/platform/issues/4993

The key class is `SierraPhysicalItemOrder` – see the comment on this object to understand the problem I'm trying to solve here. This gets us different ordering for ~10k items (different is likely better).

### CODE REVIEW

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/data/SierraPhysicalItemOrder.scala [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  *
+  *     Thing you can request
+  *     Please request top item
+  *     Another thing you can request
+  *     Please request top item
+  *
+  * This "Please request top item" occurs when two items have been bound together
+  * (e.g. on the same frame) and they have to be requested together.
+  *
+  * This ordering is stored in a part of Sierra we can't access through the
```
Comment (jamesgorrie) [inline pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/data/SierraPhysicalItemOrder.scala:23]: > we can't access

Do we have a channel to request this?

**Review by jamesgorrie — APPROVED** [review]

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/data/SierraPhysicalItemOrder.scala (hunk 1 above)
Comment (alexwlchan) [reply pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/data/SierraPhysicalItemOrder.scala:23]: Yeah, it's tracked by a couple of other GitHub tickets.

Comment (jamesgorrie) [reply pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/data/SierraPhysicalItemOrder.scala:23]: Do you have a link to that API?

Comment (alexwlchan) [reply pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/data/SierraPhysicalItemOrder.scala:23]: https://github.com/wellcomecollection/platform/issues/4102#issuecomment-608341955

That's on a ticket linked to from the ticket in this PR description.

**Pull request merged**